pypowsybl/network/impl/network.py

```diff
@@ -306,6 +306,39 @@ def get_elements(self, element_type: ElementType, all_attributes: bool = False,
             result = result[attributes]
         return result
 
+    def get_sub_networks(self, all_attributes: bool = False, attributes: List[str] = None,
+                         **kwargs: ArrayLike) -> DataFrame:
+        """
+        Get a dataframe of sub networks
```

Comment:
the dataframe just lists the name of the subnetwork ?

Reply:
Yes, there is no column (maybe one day), just the index is the ID of the sub networks